wiki/Installation/en.md

```diff
@@ -0,0 +1,31 @@
+# Installation
+
+To begin playing on Titanic! you can head over to the [downloads page](https://osu.lekuru.xyz/download/) and pick any client from as old as 2007 up to 2014.
+
+![](https://i.ibb.co/WGqryjw/Screenshot-from-2024-05-27-17-53-40.png)
+
+After it finishes installing, all you have to do is extract the folder and double-click `osu!.exe`.
+
+There is a chance that Windows Defender may block it, since the game is unsigned (a certificate for that would cost 200$).
+
+# Registering an account
+
+To register an account you can go to the [registration page](https://osu.lekuru.xyz/account/register) by clicking on "I'm New!" (**Remember you are only allowed to have *ONE* account EVER!**)
+
+![](https://i.ibb.co/fvdRBrG/Screenshot-from-2024-05-27-18-29-27.png)
+
+After you create your account you will be prompted to verify your email:
+
+![](https://i.ibb.co/rQcgSKH/Screenshot-from-2024-05-27-18-39-41.png)
+
+If you want to use the bot on our [Discord](https://discord.gg/3VeNPgDUrK) all you have to do is type `!link <username>` and you will be sent a verification code in-game.
+
+![](https://i.ibb.co/X7C6yJw/Screenshot-from-2024-05-27-19-10-45.png)
+
+![](https://i.ibb.co/YcSjzyS/Screenshot-from-2024-05-27-19-11-33.png)
+
+![](https://i.ibb.co/7SWp07k/Screenshot-from-2024-05-27-19-13-09.png[)
+
+# Linux Installation
+
```

Comment:
```suggestion
# Installation

To begin playing on Titanic! you can head over to the [downloads page](https://osu.lekuru.xyz/download/) and pick any client from as old as 2007 up to 2014.

![](https://i.ibb.co/WGqryjw/Screenshot-from-2024-05-27-17-53-40.png)

After it finishes installing, all you have to do is extract the folder and double-click `osu!.exe`.

There is a chance that Windows Defender may block it, since the game is unsigned (a certificate for that would cost 200$).

## Registering an account

To register an account you can go to the [registration page](https://osu.lekuru.xyz/account/register) by clicking on "I'm New!" (**Remember you are only allowed to have *ONE* account EVER!**)

![](https://i.ibb.co/fvdRBrG/Screenshot-from-2024-05-27-18-29-27.png)

After you create your account you will be prompted to verify your email:

![](https://i.ibb.co/rQcgSKH/Screenshot-from-2024-05-27-18-39-41.png)

## Discord Bot

If you want to use the bot on our [Discord](https://discord.gg/3VeNPgDUrK) all you have to do is type `!link <username>` and you will be sent a verification code in-game.

![](https://i.ibb.co/X7C6yJw/Screenshot-from-2024-05-27-19-10-45.png)

![](https://i.ibb.co/YcSjzyS/Screenshot-from-2024-05-27-19-11-33.png)

![](https://i.ibb.co/7SWp07k/Screenshot-from-2024-05-27-19-13-09.png[)

## Linux Installation

```

Reply:
Changed some heading here